### DISCUSSION TITLE
I can't deploy new features in my new project

### DISCUSSION THREAD

**Discussion by bcolinad** [vercel/community#2985] [Help]
### Page to Investigate

https://innovationcore.cl/

### Steps to Reproduce

I am just committing and pushing the changes but Vercel does not deploy the changes.

**Comment by amyegan** [comment]
Hi @bcolinad. Do you see any error information in the [Build Logs](https://vercel.com/docs/concepts/deployments/logs) or [Runtime Logs](https://vercel.com/docs/concepts/observability/runtime-logs)?
